### ISSUE TITLE
Option to set the rights with AND or OR

### ISSUE THREAD

**Issue by Dostoi** [GM-Alex/user-access-manager#144]
Right now you get the reading and writing right if you're member of only one of the group tagged on a page. It would be great to have an option that allow to choose if being in one of the group of a page is enough to be given the right or if the user need to be member of all the group, and for both right.

For example, I tagged pages with several groups and only want one of them to be able to write but all of them to read it. Being able to set the reading right as being given when you're member of one of the group and the writing right to be given when you're a member of all of the group, would do it easily.

In short, I want to be able to choose if the reading right is given with a AND or OR and the same for the writing right.

Thanks in advance !

**Comment by FelixDeCarbonie** [comment]
Hi ! 
I am looking for the same kind of feature.
Maybe another way to implement it would be to add a "write access" option named "none", in plus than "only group users" and "all".
That way, you can have content with any access for basic users, reading access for users for one or more groups and writing access for a part of them.

Thanks a lot for the plugin and its support !

**Comment by Dostoi** [comment]
Thanks a lot for the quick implementation !

**Comment by FelixDeCarbonie** [comment]
\o/ Wonderful !
